packages/app-rfi/src/components/stepper/RfiStepper.js

```diff
@@ -27,28 +30,41 @@ const RfiStepper = props => {
   const [email, setEmail] = useState("");
   const section = mapSections[step];
 
+  const {
+    campus,
+    studentType,
+    areaOfInterest,
+    programOfInterest,
+    programOfInterestOptional,
+    isCertMinor,
+    country,
+    stateProvince,
+    successMsg,
+    test,
+    dataSourceDegreeSearch,
+  } = useRfiContext();
+
   useEffect(() => {
     const fetchData = async () => {
       // If configured as a cert or minor and we have a program of interest, look
       // up the program's email address and set it in state for use in
       // isCertMinor render below.
-      const { dataSourceDegreeSearch, isCertMinor, programOfInterest } = props;
-      if (isCertMinor && programOfInterest) {
-        const serviceUrl =
-          `${dataSourceDegreeSearch}` +
-          `?init=false&method=findDegreeByAcadPlan&acadPlan=${programOfInterest}` +
-          `&fields=AcadPlan,EmailAddr&program=graduate&cert=true`;
 
-        const res = await fetch(serviceUrl)
-          .then(response => response.json())
-          .catch(error => new Error(error));
-        // Structure of response: response.programs[0].EmailAddr
-        if (res.programs) {
-          const {
-            programs: [{ EmailAddr }],
-          } = res;
-          setEmail(EmailAddr);
-        }
+      // Do we need this isCertMinor prop, or can we read the data to decide
```

Comment:
A good question. With the new service it may be easier to determine reliably when something is a cert or minor...

Reply:
@davidornelas11 UGCM works for undergrad certs and minors (the UG), but have you confirmed that with the grad certs? This is jogging my memory. I believe it was the lack of reliable flag for grad certs that led to the current business logic. cc @scott-williams-az